lib/install/management/dispatcher.go

```diff
@@ -294,19 +295,52 @@ func (d *Dispatcher) opManager(ctx context.Context, vch *vm.VirtualMachine) (*gu
 	return processManager, nil
 }
 
+// opManagerWait polls for state of the process with the given pid, waiting until the process has completed.
+// The pid param must be one returned by ProcessManager.StartProgram.
+func (d *Dispatcher) opManagerWait(ctx context.Context, pm *guest.ProcessManager, auth types.BaseGuestAuthentication, pid int64) (*types.GuestProcessInfo, error) {
+	pids := []int64{pid}
+	deadline, cancel := context.WithDeadline(ctx, time.Now().Add(time.Second*30))
```

Comment:
Just picked it, but suppose I could just remove that since the ctx param already has a deadline set by vic-machine.